Home.md

```diff
@@ -1,40 +1,51 @@
+
+### d:swarm *Streaming* Version
+
+The d:swarm *streaming* version offers fast processing of large amounts data and is sufficient for many scenarios. You may already use it today to start with as long as the work on the full *DataHub* version of D:SWARM continues. Unlike in the *DataHub* version this does not allow for versioning/archiving. See how  [[SLUB Dresden employs d:swarm | D:SWARM Use Case at SLUB Dresden]] for transforming and integrating bibliographic data sources. (Currently, streaming the export is implemented for XML, RDF will be added on demand.) 
+
+### d:swarm *DataHub* Version
+
+Archiving versions of the transformed data is only possible with the *DataHub* version of d:swarm, which is also the basis for [[upcoming functionality | Outlook]] such as deduplication, frbrization and other data quality improvements. While many steps into this direction have been taken, challenges remain with respect to scalability for very large datasets. See [this blog post](http://www.slub-dresden.de/blog-post-on-challenges-property-graph-handling).
+
+<!--- source code for image currently stored by jan -->
+[[img/dswarm-usage-variants.png]]
 
 ### ... and behind the scenes
 
-As shown below, the overall architecture consists of two major parts: the Backoffice [[front end]] and the [[back end]]. The back end consists of 4 modules: 
+As shown below, the overall architecture consists of three major parts: the [BackOffice web application](http://demo.dswarm.org), the [TPU](https://github.com/dswarm/task-processing-unit-for-dswarm), and the [[back end]]. The back end, in turn, consists of three modules: 
  * a [[controller]] module that controls the program flow and provides a HTTP API
- * a [[converter]] that encapsulates [Metafacture](https://github.com/culturegraph/metafacture-core) to transform data
+ * a [[converter]] that encapsulates [metafacture](https://github.com/culturegraph/metafacture-core) to transform data
  * and a [[persistence]] layer to access the [[metadata repository|Glossary#metadata-repository]] (currently a relational database; MySQL) and the [[data hub|Glossary#data-hub]] (currently a graph database; [Neo4j](http://www.neo4j.org)).
- 
-End users, e.g., system librarians, usually interact with the [Backoffice we application](http://demo.dswarm.org) and may use the [[data hub browser|Graph-Exploration]] to inspect or visualize the data stored by d:swarm.
 
-[![an image of the architecture overview](https://raw.githubusercontent.com/wiki/dswarm/dswarm-documentation/img/architecture.png)](https://raw.githubusercontent.com/wiki/dswarm/dswarm-documentation/img/architecture.png "Architecture Overview")

```

Comment:
I'll add a note to the respective tickets that we need to reenable the linking to larger image versions, once the wiki is merged.